recipe/conda_build_config.yaml

```diff
@@ -56,7 +56,7 @@ cuda_compiler:
 cuda_compiler_version:
   - None
   - 11.8                       # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
-  - 12.0                       # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
+  - 12.6                       # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
```

Comment:
As noted previously, this won't work for `linux_ppc64le`, which stops at CUDA 12.4. So that will need to be broken out